src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -188,7 +188,9 @@ private void ProcessPublishTestResultsCommand(IExecutionContext context, Diction
                         isAutomated: true,
                         buildId: runContext != null ? runContext.BuildId : 0,
                         buildFlavor: runContext != null ? runContext.Configuration : string.Empty,
-                        buildPlatform: runContext != null ? runContext.Platform : string.Empty
+                        buildPlatform: runContext != null ? runContext.Platform : string.Empty,
+                        releaseUri: runContext != null ? runContext.ReleaseUri : null,
+                        releaseEnvironmentUri: runContext != null ? runContext.ReleaseEnvironmentUri : null
```

Comment:
copied from existing code https://github.com/Microsoft/vsts-agent/blob/master/src/Agent.Worker/TestResults/JunitResultReader.cs

the reason why we set empty for Build variable is there's an issue at TCM side where they get an exception if you pass them as null. but not for release variables.